packages/cashier/src/components/empty-states/index.ts

```diff
@@ -0,0 +1,57 @@
+import EmptyStateCashierLockedStatus from './empty-state-cashier-locked-status';
+import EmptyStateCashierLockedSystemMaintenance from './empty-state-cashier-locked-system-maintenance';
+import EmptyStateCryptoLockedSystemMaintenance from './empty-state-crypto-locked-system-maintenance';
+import EmptyStateDepositLockedAskFixDetails from './empty-state-deposit-locked-ask-fix-details';
+import EmptyStateDepositLockedSystemMaintenance from './empty-state-deposit-locked-system-maintenance';
+import EmptyStateDisabledStatus from './empty-state-disabled-status';
+import EmptyStateDocumentsExpired from './empty-state-documents-expired';
+import EmptyStateIdentityVerificationNeeded from './empty-state-identity-verification-needed';
+import EmptyStateNoResidence from './empty-state-no-residence';
+import EmptyStateNoWithdrawalOrTradingStatus from './empty-state-no-withdrawal-or-trading-status';
+import EmptyStateOnlyPAWithdrawalsAllowedStatus from './empty-state-only-pa-withdrawals-allowed-status';
+import EmptyStateSelfExclusion from './empty-state-self-exclusion';
+import EmptyStateUnwelcomeStatus from './empty-state-unwelcome-status';
+import EmptyStateWithdrawServiceUnavailableForPA from './empty-state-withdraw-service-unavailable-for-pa';
+import EmptyStateWithdrawalLockedAskAuthenticate from './empty-state-withdrawal-locked-ask-authenticate';
+import EmptyStateWithdrawalLockedAskFixDetails from './empty-state-withdrawal-locked-ask-fix-details';
+import EmptyStateWithdrawalLockedFinancialAssessmentRequired from './empty-state-withdrawal-locked-financial-assessment-required';
+import EmptyStateWithdrawalLockedStatus from './empty-state-withdrawal-locked-status';
+import EmptyStateWithdrawalLockedSystemMaintenance from './empty-state-withdrawal-locked-system-maintenance';
+
+export {
+    EmptyStateAskCurrency,
+    EmptyStateAskFinancialRiskApproval,
+    EmptyStateAskSelfExclusionMaxTurnoverSet,
+    EmptyStateAskTINInformation,
+    EmptyStateAskUKFundsProtection,
+    EmptyStateCashierLockedAskAuthenticate,
+    EmptyStateCashierLockedAskFixDetails,
+    EmptyStateCashierLockedFinancialAssessmentRequired,
+    EmptyStateCashierLockedStatus,
+    EmptyStateCashierLockedSystemMaintenance,
+    EmptyStateCryptoLockedSystemMaintenance,
+    EmptyStateDepositLockedAskFixDetails,
+    EmptyStateDepositLockedSystemMaintenance,
+    EmptyStateDisabledStatus,
+    EmptyStateDocumentsExpired,
+    EmptyStateIdentityVerificationNeeded,
+    EmptyStateNoResidence,
+    EmptyStateNoWithdrawalOrTradingStatus,
+    EmptyStateOnlyPAWithdrawalsAllowedStatus,
+    EmptyStateSelfExclusion,
+    EmptyStateUnwelcomeStatus,
+    EmptyStateWithdrawServiceUnavailableForPA,
+    EmptyStateWithdrawalLockedAskAuthenticate,
+    EmptyStateWithdrawalLockedAskFixDetails,
+    EmptyStateWithdrawalLockedFinancialAssessmentRequired,
+    EmptyStateWithdrawalLockedStatus,
+    EmptyStateWithdrawalLockedSystemMaintenance,
```

Comment:
I think maybe it would be better organized if moving the `empty-states` folder inside the main `empty-state` folder, as these are all related to the `empty-state` component.

```
components/empty-state
    - states/
        - <all states components>
    - index.ts
    - empty-state.tsx
    - empty-state.scss
```

Reply:
@hamid-deriv I would rather keep them separated for better maintainability, Let's say later we want to move the `empty-state` component to the shared `components` package, Then all we needed to do was to change their import wherever it’s been used and nothing more.

Let’s hear the team's thought on this 🙇🏻

---

@Bahar-FS @george-usynin-binary What do you guys think? Can we move them to the `empty-state` folder itself as Hamid suggested? Please do let me know to update the code 🙇🏻

Reply:
hmm, 🤔  if these states related to `cashier-locked` component maybe we can move `empty-states` folder to `cashier-locked` folder (and also rename `empty-states` to e.g. `cashier-locked-states`). in `cashier-locked-states` we can import EmptyState like `import EmptyState from 'Components/empty-state';`
something like this
```
components/cashier-locked
    - cashier-locked
    - cashier-locked-states/
        - <all states components>
    - index.ts
```

Reply:
I agree with @george-usynin-binary solution. :+1:

Reply:
@george-usynin-binary Makes sense, Thanks for the suggestion, Will update the code 🙇🏻